diff --git a/src/plugins/bugzilla.conf b/src/plugins/bugzilla.conf
index 18eba5a..901fb0e 100644
--- a/src/plugins/bugzilla.conf
+++ b/src/plugins/bugzilla.conf
@@ -1,5 +1,5 @@
 # Bugzilla URL
-BugzillaURL = https://bugzilla.redhat.com/
+BugzillaURL = https://qa.mandriva.com/
 # yes means that ssl certificates will be checked
 SSLVerify = yes
 # your login has to exist, if you don have any, please create one
--- ./src/plugins/report_Bugzilla.xml.addmdv	2012-04-26 18:16:16.327775480 -0300
+++ ./src/plugins/report_Bugzilla.xml	2012-04-26 18:21:04.389649777 -0300
@@ -153,51 +153,51 @@
             <description xml:lang="uk">Адреса сервера Bugzilla</description>
             <description xml:lang="zh_CN">Bugzilla 服务器地址</description>
             <description xml:lang="zh_TW">Bugzilla </description>
-            <default-value>https://bugzilla.redhat.com</default-value>
-            <note-html>You can create bugzilla.redhat.com account &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
-            <note-html xml:lang="as">আপুনি bugzilla.redhat.com একাওন্ট সৃষ্টি কৰিব পাৰিব &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
-            <note-html xml:lang="bg">Можете да създадете bugzilla.redhat.com акаунт <a href="https://bugzilla.redhat.com/createaccount.cgi">ето тук</a></note-html>
-            <note-html xml:lang="bn_IN">&lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;-এ bugzilla.redhat.com অ্যাকাউন্ট তৈরি করা যাবে</note-html>
-            <note-html xml:lang="ca">Podeu crear un compte de bugzilla.redhat.com &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
-            <note-html xml:lang="cs">Můžete vytvořit bugzilla.redhat.com účet &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
-            <note-html xml:lang="de">Sie können einen Account für bugzilla.redhat.com erstellen <a href="https://bugzilla.redhat.com/createaccount.cgi">here</a></note-html>
-            <note-html xml:lang="en_GB">You can create bugzilla.redhat.com account &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
-            <note-html xml:lang="es">Puede crear la cuenta bugzilla.redhat.com&lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;aquí&lt;/a&gt;
+            <default-value>https://qa.mandriva.com</default-value>
+            <note-html>You can create qa.mandriva.com account &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
+            <note-html xml:lang="as">আপুনি qa.mandriva.com একাওন্ট সৃষ্টি কৰিব পাৰিব &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
+            <note-html xml:lang="bg">Можете да създадете qa.mandriva.com акаунт <a href="https://qa.mandriva.com/createaccount.cgi">ето тук</a></note-html>
+            <note-html xml:lang="bn_IN">&lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;-এ qa.mandriva.com অ্যাকাউন্ট তৈরি করা যাবে</note-html>
+            <note-html xml:lang="ca">Podeu crear un compte de qa.mandriva.com &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
+            <note-html xml:lang="cs">Můžete vytvořit qa.mandriva.com účet &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
+            <note-html xml:lang="de">Sie können einen Account für qa.mandriva.com erstellen <a href="https://qa.mandriva.com/createaccount.cgi">here</a></note-html>
+            <note-html xml:lang="en_GB">You can create qa.mandriva.com account &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
+            <note-html xml:lang="es">Puede crear la cuenta qa.mandriva.com&lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;aquí&lt;/a&gt;
 
     * details
     * suggestions
 
 	
 other
-The following url is either missing from the translation or has been translated: 'https://bugzilla.redhat.com/createaccount.cgi'.
+The following url is either missing from the translation or has been translated: 'https://qa.mandriva.com/createaccount.cgi'.
 
 copy source</note-html>
-            <note-html xml:lang="fi">Voit luoda bugzilla.redhat.com-tilin &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;täällä&lt;/a&gt;</note-html>
-            <note-html xml:lang="fr">Vous pouvez créer un compte bugzilla.redhat.com &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
-            <note-html xml:lang="gu">તમે bugzilla.redhat.com ખાતાને બનાવી શકો છો &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
-            <note-html xml:lang="hi">आप bugzilla.redhat.com खाता बना सकते हैं &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
-            <note-html xml:lang="hu">Készíthet bugzilla.redhat.com felhasználói azonosítót &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;itt&lt;/a&gt;</note-html>
-            <note-html xml:lang="it">È possibile creare un account bugzilla.redhat.com &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt; </note-html>
-            <note-html xml:lang="ja">bugzilla.redhat.com アカウントを作成できます。&lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;　</note-html>
-            <note-html xml:lang="kn">ನೀವು bugzilla.redhat.com ಖಾತೆಯನ್ನು &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;ಇಲ್ಲಿ &lt;/a&gt; ರಚಿಸಬಹುದು</note-html>
-            <note-html xml:lang="ko">&lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;여기&lt;/a&gt;에서 bugzilla.redhat.com 계정을 생성할 수 있습니다 </note-html>
-            <note-html xml:lang="ml">നിങ്ങള്‍ക്കു് &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;-ല്‍ bugzilla.redhat.com അക്കൌണ്ട് തയ്യാറാക്കാം</note-html>
-            <note-html xml:lang="mr">तुम्ही bugzilla.redhat.com खाते &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;येथे निर्माण करू शकता&lt;/a&gt;</note-html>
-            <note-html xml:lang="nl">Je kunt &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;hier&lt;/a&gt; een bugzilla.redhat.com account aanmken</note-html>
-            <note-html xml:lang="or">ଆପଣ bugzilla.redhat.com ଖାତା ନିର୍ମାଣ କରିପାରିବେ &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
-            <note-html xml:lang="pa">ਤੁਸੀਂ bugzilla.redhat.com ਅਕਾਊਂਟ ਬਣਾ ਸਕਦੇ ਹੋ &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
-            <note-html xml:lang="pl">Można utworzyć konto w witrynie bugzilla.redhat.com <a href="https://bugzilla.redhat.com/createaccount.cgi">pod tym adresem</a></note-html>
-            <note-html xml:lang="pt">Pode criar uma conta bugzilla.redhat.com &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;aqui&lt;/a&gt;</note-html>
-            <note-html xml:lang="pt_BR">Você pode criar uma conta no bugzilla.redhat.com account &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
-            <note-html xml:lang="ru">&lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;Создать&lt;/a&gt; учётную запись Bugzilla.</note-html>
-            <note-html xml:lang="sk">bugzilla.redhat.com účet je možné vytvoriť na &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;tejto adrese&lt;/a&gt;</note-html>
-            <note-html xml:lang="sv">Du kan skapa ett konto på bugzilla.redhat.com &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;här&lt;/a&gt;</note-html>
-            <note-html xml:lang="ta">bugzilla.redhat.com account &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt; ஐ நீங்கள் உருவாக்கலாம் </note-html>
-            <note-html xml:lang="te">మీరు bugzilla.redhat.com ఖాతా సృష్టించవచ్చు &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
-            <note-html xml:lang="tr">bugzilla.redhat.com hesabı yaratabilirsiniz &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
-            <note-html xml:lang="uk">Створити обліковий запис bugzilla.redhat.com можна &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;тут&lt;/a&gt;</note-html>
-            <note-html xml:lang="zh_CN">您可以在 &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt; 创建 bugzilla.redhat.com 账户</note-html>
-            <note-html xml:lang="zh_TW">您可建立 bugzilla.redhat.com 帳號 &lt;a href="https://bugzilla.redhat.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
+            <note-html xml:lang="fi">Voit luoda qa.mandriva.com-tilin &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;täällä&lt;/a&gt;</note-html>
+            <note-html xml:lang="fr">Vous pouvez créer un compte qa.mandriva.com &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
+            <note-html xml:lang="gu">તમે qa.mandriva.com ખાતાને બનાવી શકો છો &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
+            <note-html xml:lang="hi">आप qa.mandriva.com खाता बना सकते हैं &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
+            <note-html xml:lang="hu">Készíthet qa.mandriva.com felhasználói azonosítót &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;itt&lt;/a&gt;</note-html>
+            <note-html xml:lang="it">È possibile creare un account qa.mandriva.com &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt; </note-html>
+            <note-html xml:lang="ja">qa.mandriva.com アカウントを作成できます。&lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;　</note-html>
+            <note-html xml:lang="kn">ನೀವು qa.mandriva.com ಖಾತೆಯನ್ನು &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;ಇಲ್ಲಿ &lt;/a&gt; ರಚಿಸಬಹುದು</note-html>
+            <note-html xml:lang="ko">&lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;여기&lt;/a&gt;에서 qa.mandriva.com 계정을 생성할 수 있습니다 </note-html>
+            <note-html xml:lang="ml">നിങ്ങള്‍ക്കു് &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;-ല്‍ qa.mandriva.com അക്കൌണ്ട് തയ്യാറാക്കാം</note-html>
+            <note-html xml:lang="mr">तुम्ही qa.mandriva.com खाते &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;येथे निर्माण करू शकता&lt;/a&gt;</note-html>
+            <note-html xml:lang="nl">Je kunt &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;hier&lt;/a&gt; een qa.mandriva.com account aanmken</note-html>
+            <note-html xml:lang="or">ଆପଣ qa.mandriva.com ଖାତା ନିର୍ମାଣ କରିପାରିବେ &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
+            <note-html xml:lang="pa">ਤੁਸੀਂ qa.mandriva.com ਅਕਾਊਂਟ ਬਣਾ ਸਕਦੇ ਹੋ &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
+            <note-html xml:lang="pl">Można utworzyć konto w witrynie qa.mandriva.com <a href="https://qa.mandriva.com/createaccount.cgi">pod tym adresem</a></note-html>
+            <note-html xml:lang="pt">Pode criar uma conta qa.mandriva.com &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;aqui&lt;/a&gt;</note-html>
+            <note-html xml:lang="pt_BR">Você pode criar uma conta no qa.mandriva.com account &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
+            <note-html xml:lang="ru">&lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;Создать&lt;/a&gt; учётную запись Bugzilla.</note-html>
+            <note-html xml:lang="sk">qa.mandriva.com účet je možné vytvoriť na &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;tejto adrese&lt;/a&gt;</note-html>
+            <note-html xml:lang="sv">Du kan skapa ett konto på qa.mandriva.com &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;här&lt;/a&gt;</note-html>
+            <note-html xml:lang="ta">qa.mandriva.com account &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt; ஐ நீங்கள் உருவாக்கலாம் </note-html>
+            <note-html xml:lang="te">మీరు qa.mandriva.com ఖాతా సృష్టించవచ్చు &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
+            <note-html xml:lang="tr">qa.mandriva.com hesabı yaratabilirsiniz &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
+            <note-html xml:lang="uk">Створити обліковий запис qa.mandriva.com можна &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;тут&lt;/a&gt;</note-html>
+            <note-html xml:lang="zh_CN">您可以在 &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt; 创建 qa.mandriva.com 账户</note-html>
+            <note-html xml:lang="zh_TW">您可建立 qa.mandriva.com 帳號 &lt;a href="https://qa.mandriva.com/createaccount.cgi"&gt;here&lt;/a&gt;</note-html>
         </option>
         <option type="text" name="Bugzilla_Login">
             <label>User name</label>
@@ -488,4 +488,4 @@ copy source</note-html>
             </option>
         </advanced-options>
     </options>
-</event>
\ No newline at end of file
+</event>
