--- libreport-2.9.3/src/lib/ureport.c.omv~	2017-12-31 04:27:47.608663859 +0100
+++ libreport-2.9.3/src/lib/ureport.c	2017-12-31 04:27:54.671676963 +0100
@@ -609,7 +609,7 @@ ureport_server_response_from_reply(post_
 
     json_object *const json = json_tokener_parse(post_state->body);
 
-    if (is_error(json))
+    if (json == NULL)
     {
         error_msg(_("Unable to parse response from ureport server at '%s'"), config->ur_url);
         log_notice("%s", post_state->body);
