diff --git a/src/include/internal_libreport.h b/src/include/internal_libreport.h
index 95b600b..d99d84c 100644
--- a/src/include/internal_libreport.h
+++ b/src/include/internal_libreport.h
@@ -662,6 +662,7 @@ bool make_dir_recursive(char *dir, mode_t dir_mode);
 #define FILENAME_BINARY       "binary"
 #define FILENAME_CMDLINE      "cmdline"
 #define FILENAME_COREDUMP     "coredump"
+#define FILENAME_CGROUP       "cgroup"
 #define FILENAME_BACKTRACE    "backtrace"
 #define FILENAME_MAPS         "maps"
 #define FILENAME_SMAPS        "smaps"
diff --git a/src/lib/problem_data.c b/src/lib/problem_data.c
index 2a7835a..536f7a1 100644
--- a/src/lib/problem_data.c
+++ b/src/lib/problem_data.c
@@ -187,6 +187,7 @@ static const char *const editable_files[] = {
     FILENAME_OPEN_FDS   , /* user might want to hide sensitive file names */
     //FILENAME_LIMITS     ,
     FILENAME_CMDLINE    ,
+    //FILENAME_CGROUP     ,
     //FILENAME_COREDUMP   ,
     FILENAME_BACKTRACE  ,
     FILENAME_MAPS       ,
